contract-map.json

```diff
@@ -1,4 +1,11 @@
 {
+   "0xC7aDb7a340F6a9bfcEdEa0D1AECa81584E394913": {
+    "name": "CNFT",
+    "logo": "colournft.svg",
```

Comment:
Logo should have less than 10kB - this one is to big

Reply:
Already change the logo to fulfill your requirements